keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -54,17 +55,18 @@ class ResourceActuator(
 
   suspend fun <T : ResourceSpec> checkResource(resource: Resource<T>) {
     withTracingContext(resource) {
+      val coid = UUID.randomUUID().toString() // coroutine id for log messages to help debug #951
       val id = resource.id
       val plugin = handlers.supporting(resource.kind)
 
       if (actuationPauser.isPaused(resource)) {
-        log.debug("Actuation for resource {} is paused, skipping checks", id)
+        log.debug("Actuation for resource {} is paused, skipping checks [$coid]", id)
```

Comment:
One thought - if you want to group all the messages that have a coid in them together, you might want each log message to be `[coid: $coid]` so it's easy to search for all of them. If you don't want to do that, then ignore me.

Reply:
My plan is to find the `"Resource check for $id failed [$coid]" ` log message (which always appears when instances go bad), and then just search for the coid in that message to get the remaining relevant messages.